### PR TITLE
Create digital-storefront.md

### PR DESCRIPTION
**Pull request by thelifeandtimes**
for review from @tacryt-socryp.

### CODE REVIEW

File: content/grants/digital-storefront.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- Jimmy Takeria is a big calisthenics guy who teaches other guys how to get jacked. He makes informational videos and posts them to YouTube as well as running a Discord community about calisthenics. Jimmy doesn't like YouTube and Discord's new misinformation policies because he has unconventional ideas about nutrition. Jimmy decides to move his community off of Discord and into Groups on Urbit, and sets up a Studio webstore where he sells access to his latest workout videos and cutting edge nutritional info on a subscription basis. He's satisfied that he can manage his entire online business using Urbit.
+
+# Milestones and Compensation
```
Comment (tacryt-socryp) [inline content/grants/digital-storefront.md:72]: Let's push to beginning of July to be realistic.

File: content/grants/digital-storefront.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+A new `%magic` agent will be created to handle content access for digital subscriptions and one-time content sales. This agent is currently in progress and maintains state regarding users, what those users have access to, and supplies a fully-featured magic link login system utilizing HTTP cookies and sessions for accessing webpages that are behind payment gates.
+
+Current Agent Improvements:
+The `%pipe` agent will be expanded to handle publishing a new type of site template, the “store template”.
+
+The `%nmi` agent will be expanded to handle recurring subscriptions.
+
+The %magic agent will keep track of paid content and who may access it (either a @p or an email address). When a user wants to access paid content, Magic will send a “magic link” which will authenticate the user and allow them access to the underlying content. Magic links will expire after they are used, and login cookies will expire after a week.
```
Comment (tacryt-socryp) [inline content/grants/digital-storefront.md]: %magic has been renamed to %auth

File: content/grants/digital-storefront.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Subscriptions are an increasingly popular way for people to support creators they like and to receive access to digital services. Creators have a strong incentive to own their content and list of customers, as it often constitutes their entire income. Urbit's unique architecture is a perfect fit for serving both of these needs.
+
+# Implementation
+## New Agents:
+Implement a new `%shop` generic store agent as a rewrite of the %naive-market planet sale agent. Shop will keep track of product listings, prices, inventories, metadata (descriptions, images, and categories), and records of sales for three types of products: planets, subscriptions, and one-time content purchases.
+
+A new `%magic` agent will be created to handle content access for digital subscriptions and one-time content sales. This agent is currently in progress and maintains state regarding users, what those users have access to, and supplies a fully-featured magic link login system utilizing HTTP cookies and sessions for accessing webpages that are behind payment gates.
```
Comment (tacryt-socryp) [inline content/grants/digital-storefront.md]: %magic is now %auth

**Review by tacryt-socryp — CHANGES_REQUESTED** [review]

File: content/grants/digital-storefront.md (hunk 1 above)
Comment (thelifeandtimes) [reply content/grants/digital-storefront.md:72]: done.

File: content/grants/digital-storefront.md (hunk 3 above)
Comment (thelifeandtimes) [reply content/grants/digital-storefront.md]: done.

**Comment by thelifeandtimes** [comment]
@mikeosborne this on is ready for your final review.

**Review by tacryt-socryp — APPROVED** [review]

**Pull request merged**